changelog/7263.removal.md

```diff
@@ -0,0 +1,29 @@
+The [Pika Event Broker](event-brokers.mdx#pika-event-broker) no longer supports
+the environment variables `RABBITMQ_SSL_CA_FILE` and `RABBITMQ_SSL_KEY_PASSWORD`.
+You can alternatively specify `RABBITMQ_SSL_CA_FILE` in the RabbitMQ connection URL as 
+described in the 
+[RabbitMQ documentation](https://www.rabbitmq.com/uri-query-parameters.html).
+
+```yaml-rasa title="endpoints.yml
+event_broker:
+ type: pika
+ url: "amqps://user:password@host?cacertfile=path_to_ca_cert&password=private_key_password"
+ queues:
+ - my_queue
+
+```
+
+Support for `RABBITMQ_SSL_KEY_PASSWORD` was removed entirely.
```

Comment:
@erohmensing Short question. Do you know if this used by any customer? 😬

Reply:
Oooh, my guess would be yes, but I can check. Is there still support for SSL for rabbit through a different setup or env var?

Reply:
Or is it just the password that is not supported?

Reply:
Sorry, have read the description now. We definitely need support for CA files as its common to use it with self-signed certs 😬

Reply:
it's just the password for the private key which is no longer supported.